code/game/turfs/simulated/outdoors/grass.dm

```diff
@@ -13,21 +12,31 @@ var/list/grass_types = list(
 		)
 	var/grass_chance = 20
 
+	var/list/grass_types = list(
+		/obj/structure/flora/ausbushes/sparsegrass,
+		/obj/structure/flora/ausbushes/fullgrass
+		)
+
+
 /turf/simulated/floor/outdoors/grass/sif
 	name = "growth"
 	icon_state = "grass_sif"
 	edge_blending_priority = 4
 	grass_chance = 0
 	var/tree_chance = 2
 
+	grass_types = list(
+		/obj/structure/flora/sif/eyes
```

Comment:
Does this actually do anything with grass_chance = 0 ?

Reply:
No, was wanting to add more grass types first before activating it.